[2020-10-22T12:46:20.541367+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.en.xlf {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/deve/inova/skeleton/api/vendor/symfony/config/Resource/FileResource.php","line":65,"trace":[{"file":"/deve/inova/skeleton/api/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2020-10-22T12:46:20.888028+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.af.xlf [] []
[2020-10-22T12:46:20.975466+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.ar.xlf [] []
[2020-10-22T12:46:21.144425+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.az.xlf [] []
[2020-10-22T12:46:21.264315+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.be.xlf [] []
[2020-10-22T12:46:21.400288+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.bg.xlf [] []
[2020-10-22T12:46:21.520113+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.bs.xlf [] []
[2020-10-22T12:46:21.605619+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.ca.xlf [] []
[2020-10-22T12:46:21.739543+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.cs.xlf [] []
[2020-10-22T12:46:21.905790+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.cy.xlf [] []
[2020-10-22T12:46:21.975709+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.da.xlf [] []
[2020-10-22T12:46:22.090006+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.de.xlf [] []
[2020-10-22T12:46:22.220656+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.el.xlf [] []
[2020-10-22T12:46:22.352295+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.es.xlf [] []
[2020-10-22T12:46:22.503659+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.et.xlf [] []
[2020-10-22T12:46:22.622878+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.eu.xlf [] []
[2020-10-22T12:46:22.731718+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.fa.xlf [] []
[2020-10-22T12:46:22.843716+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.fi.xlf [] []
[2020-10-22T12:46:22.936959+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.fr.xlf [] []
[2020-10-22T12:46:23.072940+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.gl.xlf [] []
[2020-10-22T12:46:23.184568+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.he.xlf [] []
[2020-10-22T12:46:23.305391+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.hr.xlf [] []
[2020-10-22T12:46:23.429226+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.hu.xlf [] []
[2020-10-22T12:46:23.573089+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.hy.xlf [] []
[2020-10-22T12:46:23.670733+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.id.xlf [] []
[2020-10-22T12:46:23.805321+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.it.xlf [] []
[2020-10-22T12:46:23.949711+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.ja.xlf [] []
[2020-10-22T12:46:24.097165+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.lb.xlf [] []
[2020-10-22T12:46:24.291821+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.lt.xlf [] []
[2020-10-22T12:46:24.464702+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.lv.xlf [] []
[2020-10-22T12:46:24.597824+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.mn.xlf [] []
[2020-10-22T12:46:24.701548+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.nb.xlf [] []
[2020-10-22T12:46:24.829200+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.nl.xlf [] []
[2020-10-22T12:46:24.949188+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.nn.xlf [] []
[2020-10-22T12:46:25.070332+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.no.xlf [] []
[2020-10-22T12:46:25.185115+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.pl.xlf [] []
[2020-10-22T12:46:25.337521+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.pt.xlf [] []
[2020-10-22T12:46:25.452590+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.pt_BR.xlf [] []
[2020-10-22T12:46:25.611184+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.ro.xlf [] []
[2020-10-22T12:46:25.783577+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.ru.xlf [] []
[2020-10-22T12:46:25.936936+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.sk.xlf [] []
[2020-10-22T12:46:26.097855+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.sl.xlf [] []
[2020-10-22T12:46:26.259096+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.sq.xlf [] []
[2020-10-22T12:46:26.340864+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.sr_Cyrl.xlf [] []
[2020-10-22T12:46:26.482376+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.sr_Latn.xlf [] []
[2020-10-22T12:46:26.624653+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.sv.xlf [] []
[2020-10-22T12:46:26.776074+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.th.xlf [] []
[2020-10-22T12:46:26.901918+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.tl.xlf [] []
[2020-10-22T12:46:27.151113+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.tr.xlf [] []
[2020-10-22T12:46:27.317627+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.uk.xlf [] []
[2020-10-22T12:46:27.539874+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.vi.xlf [] []
[2020-10-22T12:46:27.839658+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.zh_CN.xlf [] []
[2020-10-22T12:46:28.082643+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/validator/Resources/translations/validators.zh_TW.xlf [] []
[2020-10-22T12:46:28.271284+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/vendor/symfony/security-core/Resources/translations/security.pt_PT.xlf [] []
[2020-10-22T12:46:28.406730+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/config/routes/dev/framework.yaml [] []
[2020-10-22T12:46:30.076531+00:00] php.DEBUG: Warning: filemtime(): stat failed for /deve/projects/lib/api/config/routes/dev/framework.yaml [] []
[2020-10-22T13:11:43.263653+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2020-10-22T13:11:44.167472+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"89ea6e"},"request_uri":"http://localhost:8051/_wdt/89ea6e","method":"GET"} []
[2020-10-22T13:11:44.668993+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost:8051/")" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost:8051/\") at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2020-10-22T14:21:26.145017+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2020-10-22T14:21:27.650996+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a9884a"},"request_uri":"http://localhost:8051/_wdt/a9884a","method":"GET"} []
[2020-10-22T14:21:27.970997+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost:8051/")" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost:8051/\") at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2020-10-22T14:21:48.510389+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"a9884a"},"request_uri":"http://localhost:8051/_profiler/a9884a?panel=request","method":"GET"} []
[2020-10-22T14:21:48.880398+00:00] request.INFO: Matched route "_profiler_search_bar". {"route":"_profiler_search_bar","route_parameters":{"_route":"_profiler_search_bar","_controller":"web_profiler.controller.profiler::searchBarAction"},"request_uri":"http://localhost:8051/_profiler/search_bar?panel=request","method":"GET"} []
[2020-10-22T22:12:03.513784+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2020-10-22T22:12:05.032870+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"d0d2ff"},"request_uri":"http://localhost:8051/_wdt/d0d2ff","method":"GET"} []
[2020-10-22T22:12:05.378132+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost:8051/")" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost:8051/\") at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2020-10-22T22:16:35.071361+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2020-10-22T22:16:35.890837+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3c1d58"},"request_uri":"http://localhost:8051/_wdt/3c1d58","method":"GET"} []
[2020-10-23T01:40:07.753121+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /blump"" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /blump\" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/blump/\". at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2020-10-23T01:40:09.809168+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"53f940"},"request_uri":"http://localhost:8051/_wdt/53f940","method":"GET"} []
[2020-10-23T01:40:14.558121+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /blamp"" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /blamp\" at /deve/inova/skeleton/api/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/blamp/\". at /deve/inova/skeleton/api/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2020-10-23T01:40:15.914507+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"95fba9"},"request_uri":"http://localhost:8051/_wdt/95fba9","method":"GET"} []
